grafana/dashboards/summary/nodes.json

```diff
@@ -0,0 +1,339 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "description": "kube-state-metrics summary nodes",
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 0,
+  "hideControls": false,
+  "id": 1327,
+  "links": [],
+  "rows": [
+    {
+      "collapse": false,
+      "height": "250px",
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "k8s_monitor",
+          "fill": 1,
+          "id": 1,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "percentage": false,
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "span": 12,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "aggregator": "sum",
+              "downsampleAggregator": "avg",
+              "downsampleFillPolicy": "none",
+              "expr": "avg by(node)(kube_node_spec_unschedulable{alias=\"$alias\"}) > 0",
```

Comment:
I actually just discussed the multi replica usage with @fabxc and it turns out that scraping multiple replicas actually makes little sense especially when you want to summarize data then it shows up multiple times. So we should actually be scraping kube-state-metrics through a Service.